### PR TITLE
Add sriwedari70 and move kamigakushi to new folder

### PR DESCRIPTION
**Pull request by alabahuy**
## Description

add new board sriwedari70 dan move kamigakushi to new folder

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/jaykeeb/sriwedari70/config.h [hunk 1]
```diff
@@ -0,0 +1,20 @@
+// Copyright 2023 Reza Alif Dio (@Alabahuy)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
```
Comment (waffle87) [inline keyboards/jaykeeb/sriwedari70/config.h:5]: This file should be removed.

**Review by waffle87 — CHANGES_REQUESTED** [review]
Please also update `data/mappings/keyboard_aliases.hjson` accordingly.

File: keyboards/jaykeeb/sriwedari70/config.h (hunk 1 above)
Comment (alabahuy) [reply keyboards/jaykeeb/sriwedari70/config.h]: done

**Comment by alabahuy** [comment]
any update @waffle87 ?

**Comment by alabahuy** [comment]
do I have to close this PR and create a separate new PR?

**Comment by alabahuy** [comment]
Hey @waffle87 the conflicts already resolved

**Comment by rizalfr** [comment]
conflict has been fixed ya @waffle87

**Review by waffle87 — APPROVED** [review]

File: keyboards/jaykeeb/kamigakushi/info.json [hunk 2]
```diff
@@ -18,6 +18,7 @@
         "on_state": 1
     },
     "rgblight": {
+        "pin": "GP25",
```
Comment (fauxpark) [inline keyboards/jaykeeb/kamigakushi/info.json:21]:
```suggestion
```

File: keyboards/jaykeeb/kamigakushi/info.json [hunk 3]
```diff
@@ -36,10 +37,6 @@
         "twinkle": true
         }
     },
```
Comment (fauxpark) [inline keyboards/jaykeeb/kamigakushi/info.json:39]:
```suggestion
    },
    "ws2812": {
        "pin": "GP25",
        "driver": "vendor"
    },
```

File: keyboards/jaykeeb/kamigakushi/rules.mk [hunk 4]
```diff
@@ -0,0 +1,3 @@
+SERIAL_DRIVER = vendor
+WS2812_DRIVER = vendor
```
Comment (fauxpark) [inline keyboards/jaykeeb/kamigakushi/rules.mk:2]:
```suggestion
```

File: keyboards/jaykeeb/sriwedari70/info.json [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+        "on_state": 0
+    },
+    "encoder": {
+        "rotary": [
+            {"pin_a": "GP25", "pin_b": "GP26", "resolution": 2},
+            {"pin_a": "GP27", "pin_b": "GP28", "resolution": 2},
+        ]
+    },
+    "rgblight": {
+        "pin": "GP18",
```
Comment (fauxpark) [inline keyboards/jaykeeb/sriwedari70/info.json:32]:
```suggestion
```

File: keyboards/jaykeeb/sriwedari70/info.json [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+        "christmas": true,
+        "knight": true,
+        "rainbow_mood": true,
+        "rainbow_swirl": true,
+        "rgb_test": true,
+        "snake": true,
+        "static_gradient": true,
+        "twinkle": true
+        }
+    },
```
Comment (fauxpark) [inline keyboards/jaykeeb/sriwedari70/info.json:50]:
```suggestion
    },
    "ws2812": {
        "pin": "GP18",
        "driver": "vendor"
    },
```

File: keyboards/jaykeeb/sriwedari70/rules.mk [hunk 7]
```diff
@@ -0,0 +1,2 @@
+# This file intentionally left blank
+WS2812_DRIVER = vendor
```
Comment (fauxpark) [inline keyboards/jaykeeb/sriwedari70/rules.mk:2]:
```suggestion
```

File: keyboards/jaykeeb/sriwedari70/info.json [hunk 8]
```diff
@@ -0,0 +1,145 @@
...
+                { "matrix": [4, 0], "x": 0, "y": 4 },
+                { "matrix": [4, 1], "x": 1.25, "y": 4.25 },
+                { "matrix": [4, 2], "x": 2.25, "y": 4.25 },
+                { "matrix": [4, 3], "x": 3.25, "y": 4.25 },
+                { "matrix": [4, 4], "w": 1.5, "x": 4.5, "y": 4 },
+                { "matrix": [4, 5], "w": 1.5, "x": 6, "y": 4 },
+                { "matrix": [4, 10], "w": 7, "x": 7.5, "y": 4 },
+                { "matrix": [4, 14], "w": 1.5, "x": 14.5, "y": 4 },
+                { "matrix": [4, 15], "x": 16, "y": 4 },
+                { "matrix": [4, 16], "w": 1.5, "x": 17, "y": 4 }
```
Comment (fauxpark) [inline keyboards/jaykeeb/sriwedari70/info.json:141]:
```suggestion
                {"matrix": [0, 0], "x": 0, "y": 0},
                {"matrix": [0, 1], "x": 1, "y": 0},
                {"matrix": [0, 2], "x": 2.25, "y": 0},
                {"matrix": [0, 3], "x": 3.5, "y": 0},
                {"matrix": [0, 4], "x": 4.5, "y": 0},
                {"matrix": [0, 5], "x": 5.5, "y": 0},
                {"matrix": [0, 6], "x": 6.5, "y": 0},
                {"matrix": [0, 7], "x": 7.5, "y": 0},
                {"matrix": [0, 8], "x": 8.5, "y": 0},
                {"matrix": [0, 9], "x": 9.5, "y": 0},
                {"matrix": [0, 10], "x": 10.5, "y": 0},
                {"matrix": [0, 11], "x": 11.5, "y": 0},
                {"matrix": [0, 12], "x": 12.5, "y": 0},
                {"matrix": [0, 13], "x": 13.5, "y": 0},
                {"matrix": [0, 14], "x": 14.5, "y": 0},
                {"matrix": [0, 15], "x": 15.5, "y": 0},
                {"matrix": [0, 16], "x": 16.5, "y": 0},
                {"matrix": [1, 16], "x": 17.5, "y": 0},

                {"matrix": [1, 0], "x": 0, "y": 1},
                {"matrix": [1, 1], "x": 1, "y": 1},
                {"matrix": [1, 2], "x": 2.25, "y": 1},
                {"matrix": [1, 3], "x": 3.5, "y": 1, "w": 1.5},
                {"matrix": [1, 4], "x": 5, "y": 1},
                {"matrix": [1, 5], "x": 6, "y": 1},
                {"matrix": [1, 6], "x": 7, "y": 1},
                {"matrix": [1, 7], "x": 8, "y": 1},
                {"matrix": [1, 8], "x": 9, "y": 1},
                {"matrix": [1, 9], "x": 10, "y": 1},
                {"matrix": [1, 10], "x": 11, "y": 1},
                {"matrix": [1, 11], "x": 12, "y": 1},
                {"matrix": [1, 12], "x": 13, "y": 1},
                {"matrix": [1, 13], "x": 14, "y": 1},
                {"matrix": [1, 14], "x": 15, "y": 1},
                {"matrix": [1, 15], "x": 16, "y": 1},
                {"matrix": [2, 16], "x": 17, "y": 1, "w": 1.5},

                {"matrix": [2, 0], "x": 0, "y": 2},
                {"matrix": [2, 1], "x": 1, "y": 2},
                {"matrix": [2, 3], "x": 3.5, "y": 2, "w": 1.75},
                {"matrix": [2, 4], "x": 5.25, "y": 2},
                {"matrix": [2, 5], "x": 6.25, "y": 2},
                {"matrix": [2, 6], "x": 7.25, "y": 2},
                {"matrix": [2, 7], "x": 8.25, "y": 2},
                {"matrix": [2, 8], "x": 9.25, "y": 2},
                {"matrix": [2, 9], "x": 10.25, "y": 2},
                {"matrix": [2, 10], "x": 11.25, "y": 2},
                {"matrix": [2, 11], "x": 12.25, "y": 2},
                {"matrix": [2, 12], "x": 13.25, "y": 2},
                {"matrix": [2, 13], "x": 14.25, "y": 2},
                {"matrix": [2, 14], "x": 15.25, "y": 2},
                {"matrix": [2, 15], "x": 16.25, "y": 2, "w": 2.25},

                {"matrix": [3, 0], "x": 0, "y": 3},
                {"matrix": [3, 1], "x": 1, "y": 3},
                {"matrix": [3, 2], "x": 2.25, "y": 3.25},
                {"matrix": [3, 3], "x": 3.5, "y": 3, "w": 1.25},
                {"matrix": [3, 4], "x": 4.75, "y": 3},
                {"matrix": [3, 5], "x": 5.75, "y": 3},
                {"matrix": [3, 6], "x": 6.75, "y": 3},
                {"matrix": [3, 7], "x": 7.75, "y": 3},
                {"matrix": [3, 8], "x": 8.75, "y": 3},
                {"matrix": [3, 9], "x": 9.75, "y": 3},
                {"matrix": [3, 10], "x": 10.75, "y": 3},
                {"matrix": [3, 11], "x": 11.75, "y": 3},
                {"matrix": [3, 12], "x": 12.75, "y": 3},
                {"matrix": [3, 13], "x": 13.75, "y": 3},
                {"matrix": [3, 14], "x": 14.75, "y": 3},
                {"matrix": [3, 15], "x": 15.75, "y": 3, "w": 1.75},
                {"matrix": [3, 16], "x": 17.5, "y": 3},

                {"matrix": [4, 0], "x": 0, "y": 4},
                {"matrix": [4, 1], "x": 1.25, "y": 4.25},
                {"matrix": [4, 2], "x": 2.25, "y": 4.25},
                {"matrix": [4, 3], "x": 3.25, "y": 4.25},
                {"matrix": [4, 4], "x": 4.5, "y": 4, "w": 1.5},
                {"matrix": [4, 5], "x": 6, "y": 4, "w": 1.5},
                {"matrix": [4, 10], "x": 7.5, "y": 4, "w": 7},
                {"matrix": [4, 14], "x": 14.5, "y": 4, "w": 1.5},
                {"matrix": [4, 15], "x": 16, "y": 4},
                {"matrix": [4, 16], "x": 17, "y": 4, "w": 1.5}
```

File: keyboards/jaykeeb/sriwedari70/info.json [hunk 9]
```diff
@@ -0,0 +1,145 @@
...
+        "rows": ["GP23", "GP29", "GP15", "GP21", "GP22"]
+    },
+    "indicators": {
+        "caps_lock": "GP0",
+        "on_state": 0
+    },
+    "encoder": {
+        "rotary": [
+            {"pin_a": "GP25", "pin_b": "GP26", "resolution": 2},
+            {"pin_a": "GP27", "pin_b": "GP28", "resolution": 2},
```
Comment (fauxpark) [inline keyboards/jaykeeb/sriwedari70/info.json:28]:
```suggestion
            {"pin_a": "GP27", "pin_b": "GP28", "resolution": 2}
```

File: keyboards/jaykeeb/sriwedari70/info.json [hunk 10]
```diff
@@ -0,0 +1,145 @@
...
+        "alternating": true,
+        "breathing": true,
+        "christmas": true,
+        "knight": true,
+        "rainbow_mood": true,
+        "rainbow_swirl": true,
+        "rgb_test": true,
+        "snake": true,
+        "static_gradient": true,
+        "twinkle": true
```
Comment (fauxpark) [inline keyboards/jaykeeb/sriwedari70/info.json:48]:
```suggestion
        "animations": {
            "alternating": true,
            "breathing": true,
            "christmas": true,
            "knight": true,
            "rainbow_mood": true,
            "rainbow_swirl": true,
            "rgb_test": true,
            "snake": true,
            "static_gradient": true,
            "twinkle": true
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by alabahuy** [comment]
Thanks @waffle87 @fauxpark

**Comment by alabahuy** [comment]
any update for merged? @waffle87 @fauxpark

**Review by tzarc — APPROVED** [review]
Next time this needs to be two PRs -- as per the [PR Checklist](https://docs.qmk.fm/#/pr_checklist) each PR should be the smallest set of changes.

One PR would be the move, one PR would be the new keyboard.

Please don't repeat this next time.

**Pull request merged**